### ISSUE TITLE
Use unicode encoding instead of html entities

### ISSUE THREAD

**Issue by UngQuangHuy** [gjtorikian/mtex2MML#71]
Thank you for your contribution. Are we able to switch to use Unicode encoding instead of HTML entities?

**Comment by gjtorikian** [comment]
Yes, that should be possible. If I find the time I can make it happen, otherwise I would accept a PR!
